Terminal/README.md

```diff
@@ -0,0 +1,93 @@
+# Terminal Interface
+
+**Status**: Proposed
+
+| Name      | ID                                                                |
+| :-------- | :---------------------------------------------------------------- |
+| Terminal  | 8796536366ee21852db56dccb60bc564598b618c865fc50c8b1ab740bba128e3  |
+
+
+## Description
+
+Simple input/output. Allows to print string messages to the user and get input string from the user.
+
+## Functions
+
+Since DeBot is a smart contract then all functions work asyncronously by design. It means that they don't return anything despite the fact that all have `returns` value in their specification. Result callback will be called later and it must have arguments defined in `returns` section.
+
+`input` - prints prompt message to the user and returns string entered by user
+
+arguments:
+
+	answerId: uint32 - function id of result callback
+
+	prompt: bytes - string printed to the user and describing what to enter
+
+	multiline: bool - allow multiline text input
```

Comment:
@Keshoid do you think this argument is important to have?
For instance Surf plans to have a single UIChatInput component with an autogrow effect (to support the multiline by default).

Reply:
It allows debot developer to be sure that input string contains only 1 line.